app/src/main/java/com/mayokunadeniyi/instantweather/ui/home/HomeFragment.kt

```diff
@@ -316,4 +342,14 @@ class HomeFragment : BaseFragment() {
             ExistingPeriodicWorkPolicy.REPLACE, weatherUpdateRequest
         )
     }
+
+    private fun shareWeatherInformation(WeatherInformation:String){
```

Comment:
Hi, @ELTEGANI, thanks for this! Just a little change, could we get the parameter `WeatherInformation` to start with a small letter `weatherInformation`?

Reply:
will do

Reply:
@mayokunthefirst done